docker/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+services:
+  roop-cpu: &base
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile.cpu
+    volumes:
```

Comment:
Expose the temp and output folder 

    volumes:
      - roop/output:/home/node/roop/output
      - roop/temp:/home/node/roop/temp

Reply:
Do you really want to have them as Docker Volumes or do you want to bind-mount a directory from the host?
Bind-mounting a directory would be (the prepending `./` makes it a bind-mount):
```
services:
  roop:
    volumes:
      - ./roop/output:/home/node/roop/output
      - ./roop/temp:/home/node/roop/temp
```

Mounting a volume would be (can't have slashes in volume name):
```
services:
  roop:
    volumes:
      - output:/home/node/roop/output
      - temp:/home/node/roop/temp
volumes:
  output:
  temp:
```